src/biokbase/narrative/jobs/job.py

```diff
@@ -113,12 +122,19 @@ def __init__(self, ee2_state, extra_data=None, children=None):
         self.children = children
 
     @classmethod
-    def from_job_id(cls, job_id, extra_data=None, children=None):
-        state = cls.query_ee2_state(job_id, init=True)
-        return cls(state, extra_data=extra_data, children=children)
+    def from_job_id(
+        cls,
+        job_id: str,
+        extra_data: dict[str, Any] | None = None,
+        children: list["Job"] | None = None,
+    ):
+        state = cls.query_ee2_states([job_id], init=True)
```

Comment:
Use the plural form. Ideally I would like to get rid of the singular form to simplify the module.